docker-compose-test.yaml

```diff
@@ -0,0 +1,11 @@
+version: "3.7"
+
+services:
+  test:
+    container_name: react-app-test
+    build:
+      context: .
+      dockerfile: Dockerfile-test
+    volumes:
+      - './react-app:/usr/src/app'
+      - '/usr/src/app/node_modules'
```

Comment:
`lint` `security` and `test` are missing isn't it?

Reply:
Tried to execute it for test only,Will add lint and security